plugin/references.py

```diff
@@ -30,17 +30,17 @@ class LspSymbolReferencesCommand(LspTextCommand):
 
     capability = 'referencesProvider'
 
-    def __init__(self, view: sublime.View) -> None:
-        super().__init__(view)
-        self._picker = None  # type: Optional[LocationPicker]
-
     def run(self, _: sublime.Edit, event: Optional[dict] = None, point: Optional[int] = None) -> None:
         session = self.best_session(self.capability)
         file_path = self.view.file_name()
         pos = get_position(self.view, event, point)
         if session and file_path and pos is not None:
-            params = text_document_position_params(self.view, pos)
-            params['context'] = {"includeDeclaration": False}
+            textDocumentPositionParams = text_document_position_params(self.view, pos)
```

Comment:
Sorry for being late with replying. My OCD is telling me this variable should be snake_case :)

Reply:
I agree. :)